test/everypolitician/popolo/json_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class Everypolitician::Popolo::JsonTest < Minitest::Test
+  def test_legislative_periods
+    popolo_json = Everypolitician::Popolo::JSON.new(
+      events: [
+        { name: 'Election 1', classification: 'general election', start_date: '2014-01-01' },
+        { name: 'Term 2', classification: 'legislative period', start_date: '2015-01-01' },
+        { name: 'Term 1', classification: 'legislative period', start_date: '2010-01-01' },
+      ],
+    )
+    assert_equal 2, popolo_json.legislative_periods.size
+    assert_equal 2, popolo_json.terms.size
+    assert_equal 'Term 2', popolo_json.current_legislative_period.name
+    assert_equal 'Term 2', popolo_json.current_term.name
```

Comment:
There's a little too much going on in this test, I think. We should keep tests as much as possible to one concept per test, unless the later test would make no sense independently.

Here this is testing both that we ignore every event that's not a 'legislative period' _and_ that they're ordered correctly, which should probably each be individual tests.